orca/reconstruct.py

```diff
@@ -0,0 +1,14 @@
+from xarray import open_mfdataset, open_dataset
+
+
+def reconstruct_dataset(temp_files, outfile):
+    temp_paths = [temp.name for temp in temp_files]
+    dataset = (
+        open_mfdataset(temp_paths, combine="nested", concat_dim="time")
```

Comment:
Is it not possible to call `open_mfdataset` with a single path?

Reply:
Good point. You probably can, I will try it.